src/components/GoodsData.ts

```diff
@@ -0,0 +1,51 @@
+import goodsData from '../modules/goods';
+
+export default class GoodsData {
+  static getAllPrice(): number[] {
+    const allPrices: number[] = [];
+    for (let i = 0; i < goodsData.products.length; i++) {
+      allPrices.push(goodsData.products[i].price);
+    }
+    return allPrices;
+  }
+
+  static getMinPrice(): number {
+    const minPrice: number = Math.min(...this.getAllPrice());
+    return minPrice;
+  }
```

Comment:
you can return without creating a variable
`
static getMinPrice(): number {
    return Math.min(...this.getAllPrice());
  } 
`